gangplank/cmd/minio.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/coreos/gangplank/ocp"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+/*
+	In enviroments (such as Jenkins) were something else handles the visiualization
+	of stages, using the inherent stages in Gangplank can produced jumbled garbage.
+
+	Each Gangpplank worker requires a Minio Server running on the coordinating host/pod.
+	The `gangplank mino` command allows for starting a Minio instances and then saving
+	the configuration to allow for a shared `minio` instance.
+
+	When using a shared instance, you must use `cosa build --delay-meta-merge`.
+
+	Example start
+		nohup gangplank minio -m /tmp/minio.cfg -d /srv &
+
+	Example worker accessing minio:
+		gangplank pod -m /tmp/minio.cfg --spec test.spec
+
+	When running in minio mode, Gangplank will continue to run until it is sig{kill,term}ed.
+
+*/
+
+var cmdMinio = &cobra.Command{
+	Use:   "minio",
+	Short: "Start running a minio server",
+	Run:   runMinio,
+}
+
+var (
+	// minioCfgFile is an ocp.minioServerCfg file
+	minioCfgFile string
+)
+
+func init() {
+	cmdRoot.AddCommand(cmdMinio)
+	cmdRoot.PersistentFlags().StringVarP(&minioCfgFile, "minioCfgFile", "m", "", "location of where to create of external minio config file")
+	cmdMinio.Flags().StringVarP(&cosaWorkDir, "minioSrvDir", "d", "", "location to service minio from")
```

Comment:
minor but not necessary: `cosaWorkDir` tripped me up a bit because I kept thinking it was the `cosaWorkDir` from the `pod` sub-command.  Can we change it to `minioSrvDir`?

Reply:
I'll do a follow up on that, since this blocking 2095

Reply:
Meh, nervermind :) fixed.